### PR TITLE
Feature- Product listing page

### PR DESCRIPTION
**Pull request by Koushith**
## Description
- [x] adds product listing page

## Screenshot and Code Snippets

image

### CODE REVIEW

**Review by AurobindoGupta — APPROVED** [review]
nice work~!

File: src/pages/product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  background-color: var(--darkBlue);
+}
+
+.add-cart {
+  margin: 7px 5px;
+}
+
+.card-badge {
+  background: rgb(0, 160, 152);
+}
```
Comment (rishichourasia) [inline src/pages/product-listing/product-listing.css:95]: Could be used variables

**Review by rahul31agrawal — APPROVED** [review]
looks good to merge

**Review by rishichourasia — APPROVED** [review]
Overall code structures looks neat . Good UI  
Maybe add 1px solid border or box shadow to your product card .

**Comment by Koushith** [comment]
> Overall code structures looks neat . Good UI Maybe add 1px solid border or box shadow to your product card .

Thanks for the feedback. will reiterate in component library,

File: src/pages/product-listing/product-listing.css (hunk 1 above)
Comment (Koushith) [reply src/pages/product-listing/product-listing.css:95]: yeah, added in latest commit.

**Pull request merged**